person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] **[OPTIONAL]** It's probably a good idea to place all private methods at the end of the class, instead of defining private and public again, you simply have the public method at the top and define private. That way there is no need to write `public` again

  ```
  def can_use_services?
  ...

  private

  def of_age?
    ...
  ```